src/scratchpad/src/org/apache/poi/hsmf/datatypes/ChunkBasedPropertyValue.java

```diff
@@ -22,8 +22,8 @@ Licensed to the Apache Software Foundation (ASF) under one or more
  * TODO Provide a way to link these up with the chunks
  */
 public class ChunkBasedPropertyValue extends PropertyValue {
-    public ChunkBasedPropertyValue(MAPIProperty property, long flags, byte[] offsetData) {
-        super(property, flags, offsetData);
+    public ChunkBasedPropertyValue(MAPIProperty property, long flags, byte[] offsetData, Types.MAPIType actualType) {
```

Comment:
could you add back the old constructor? we don't remove public methods without deprecating them first